src/preset/manager.ts

```diff
@@ -1,36 +1,44 @@
-import { addons, types } from "@storybook/addons";
+import { addons } from "@storybook/addons";
+import { STORY_CHANGED, STORY_ARGS_UPDATED } from "@storybook/core-events";
+import { parsePath } from "../parsePath";
 
-import { ADDON_ID, TOOL_ID, PANEL_ID, TAB_ID } from "../constants";
-import { Tool } from "../Tool";
-import { Panel } from "../Panel";
-import { Tab } from "../Tab";
+// @ts-ignore
+import amplitude from "amplitude-js";
 
-// Register the addon
-addons.register(ADDON_ID, () => {
-  // Register the tool
-  addons.add(TOOL_ID, {
-    type: types.TOOL,
-    title: "My addon",
-    match: ({ viewMode }) => !!(viewMode && viewMode.match(/^(story|docs)$/)),
-    render: Tool,
-  });
+addons.register("storybook/amplitude", (api) => {
+  if (process.env.NODE_ENV === "production") {
```

Comment:
I don't know if this is even possible, but what do you think about this also taking an argument from the CLI as well to manually override? IDK that it'll matter much with storybook, but at my last job we'd often run into prod issues that were difficult to reproduce in local/staging envs due to hard coded prod-like conditionals.

Reply:
Hm, I'll give this some thought. Not totally sure how I'd resolve this.

Reply:
Yeah, don't let it block anything, just something I've started paying attention to.